engines/bastion_katello/app/assets/stylesheets/bastion_katello/bastion_katello.scss

```diff
@@ -26,4 +26,7 @@
     text-overflow: ellipsis;
   }
 
+  .breadcrumb {
+    margin-bottom: 0px;
```

Comment:
I personally think 10px looks better here, 0px makes it squished against what follows and also by setting this to zero we are relying on the structure below to have margin/padding which may not always be the case.